### ISSUE TITLE
Lifecycle hook replace_triggered_by is unsupported

### ISSUE THREAD

**Issue by starlightromero** [terraform-linters/tflint-ruleset-aws#372]
In Terraform 1.2.0 the lifecycle hook `replace_triggered_by` was [introduced](https://www.terraform.io/language/meta-arguments/lifecycle#replace_triggered_by). As of `tflint-ruleset-aws` version 0.16.1, this is an unsupported argument.

```bash
Installing `aws` plugin...
Installed `aws` (source: github.com/terraform-linters/tflint-ruleset-aws, version: 0.16.1)
TFLint in terraform/modules/tfe-agent-role/:
Failed to load configurations; main.tf:139,5-25: Unsupported argument; An argument named "replace_triggered_by" is not expected here.:
Error: Unsupported argument
on main.tf line 139, in resource "kubernetes_deployment" "tfc_cloud_agent":
139:     replace_triggered_by = [
An argument named "replace_triggered_by" is not expected here.
```

**Comment by starlightromero** [comment]
This is a tflint problem, not an aws ruleset issue
